### ISSUE TITLE
Print the worker's scope in the "Mocking enabled" message

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/msw#1103]
It may help in some cases of debugging to see what is the registered worker's scope. I don't think there's a way to see the worker's scope in the DevTools, it only lists "clients".

```
[MSW] Mocking enabled.
- Documentation: https://mswjs.io/docs
- Find an issue? <GITHUB_LINK>
- Scope: /
```

We may also consider printing additional information:
- The worker script location;
- The list of initial request handlers.

**Comment by Poivey** [comment]
Hi, I would like to contribute on this issue.
About the list of initial request handlers, this might be a lot of lines in the console for an "every day" usage. Do you think this more explicit worker message should be displayed if the worker is started with a new "debug" option ?

**Comment by kettanaito** [comment]
Hey, @Poivey. Great! 

> About the list of initial request handlers, this might be a lot of lines in the console for an "every day" usage. 

True. People can always print out the list of handler via [.printHandlers()](https://mswjs.io/docs/api/setup-worker/print-handlers) method, so we can skip this. Let's start with printing the worker's scope first.

**Comment by kettanaito** [comment]
## Released: v0.40.0 🎉

This has been released in v0.40.0!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v0.40.0)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/0.40.0)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
